module_utils/configuration.py

```diff
@@ -287,13 +292,23 @@ def match_filters(filter_params, obj):
         filters = params.get(ParamName.FILTERS) or {}
         if QueryParams.FILTER not in url_params[ParamName.QUERY_PARAMS] and 'name' in filters:
             # most endpoints only support filtering by name, so remaining `filters` are applied on returned objects
-            url_params[ParamName.QUERY_PARAMS][QueryParams.FILTER] = 'name:%s' % filters['name']
+            url_params[ParamName.QUERY_PARAMS][QueryParams.FILTER] = self._stringify_name_filter(filters)
 
         item_generator = iterate_over_pageable_resource(
             partial(self.send_general_request, operation_name=operation_name), url_params
         )
         return (i for i in item_generator if match_filters(filters, i))
 
+    def _stringify_name_filter(self, filters):
+        if self._system_info['version'] >= '6.4.0':
+            return "fts~%s" % filters['name']
+        return "name:%s" % filters['name']
+
+    def _fetch_system_info(self):
+        params = {"path_params": {"objId": "default"}}
```

Comment:
There is a `PATH_PARAMS_FOR_DEFAULT_OBJ` variable in `configuration.py` file. Please, reuse it.

Reply:
good catch.